test/integrations/destinations/zapier/processor/data.ts

```diff
@@ -703,4 +759,214 @@ export const data = [
       },
     },
   },
+  {
+    id: 'Test 5',
+    name: 'zapier',
+    description: 'Track call with Multiplexing',
+    scenario: 'Framework+Business',
+    feature: 'processor',
+    module: 'destination',
+    version: 'v0',
+    input: {
+      request: {
+        body: [
+          {
+            message: {
+              event: 'def',
+              userId: 'identified user id',
+              type: 'track',
+              anonymousId: 'anon-id-new',
+              timestamp: '2020-02-02T00:23:09.544Z',
+              originalTimestamp: '2020-04-17T14:42:44.724Z',
+              sentAt: '2020-04-17T14:42:44.724Z',
+            },
+            metadata: {
+              destinationId: 'destId',
+              workspaceId: 'wspId',
+            },
+            destination: {
+              DestinationDefinition: {
+                Config: {
+                  cdkV2Enabled: true,
+                },
+              },
+              Config: {
+                zapUrl: 'abcd.zap-hook',
+                trackEventsToZap: [
+                  {
+                    from: 'def',
+                    to: ['def.zap-hook', 'ghi.zap-hook'],
+                  },
+                ],
+                pageScreenEventsToZap: [{}],
+              },
+              Enabled: true,
+              Transformations: [],
+              IsProcessorEnabled: true,
+            },
+          },
+        ],
+        method: 'POST',
+      },
+    },
+    output: {
+      response: {
+        status: 200,
+        body: [
+          {
+            output: {
+              version: '1',
+              type: 'REST',
+              method: 'POST',
+              endpoint: ['def.zap-hook', 'ghi.zap-hook'],
```

Comment:
this means 2 individual calls to both endpoints, is this format common across transformer now?